downstream/modules/automation-hub/proc-using-policies-with-signed-images.adoc

```diff
@@ -10,7 +10,7 @@ When ensuring a signature is signed by a specific signature(s), the signature mu
 
 .Procedure
 
-. Click *Signature Keys* in the left menu.
+. Click btn:[Signature Keys] in the left menu.
```

Comment:
Using directional information such as left, right...is a violation. I also think this is a menu selection, suggest changing to:

In the navigation pane, select menu:Signature Keys[].